homeassistant/components/openweathermap/const.py

```diff
@@ -94,6 +95,7 @@
     ATTR_API_CLOUDS,
     ATTR_API_RAIN,
     ATTR_API_SNOW,
+    ATTR_API_PRECIPITATION_KIND,
```

Comment:
I recommend removing `MONITORED_CONDITIONS` and `FORECAST_MONITORED_CONDITIONS` and just iterate the dicts `WEATHER_SENSOR_TYPES` and `FORECAST_SENSOR_TYPES` instead.

There already seems to be a bug where the lists aren't synced with the dict keys. `ATTR_FORECAST_WIND_SPEED` isn't included in `FORECAST_SENSOR_TYPES`.

Reply:
That's putting a lot of trust in my Python skills!!! :rofl:

I haven't gone deep into the inner workings of how this integration does its thing - all I'm trying to do in this PR is add a sensor that'll let me know what kind of precipitation the aggregated 'precipitation' sensor is reporting. I can take a stab at implementing your recommendations, but should that be done as part of this PR?

Reply:
It's ok to do in a follow up. 👍